core/src/main/java/com/onewelcome/core/usecase/GetMobileAuthenticationEnrollmentStatusUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.onewelcome.core.usecase
+
+import com.github.michaelbull.result.Result
+import com.github.michaelbull.result.map
+import com.onegini.mobile.sdk.android.model.entity.UserProfile
+import com.onewelcome.core.entity.MobileAuthEnrollmentStatus
+import com.onewelcome.core.omisdk.facade.OmiSdkFacade
+import javax.inject.Inject
+
+class GetMobileAuthenticationEnrollmentStatusUseCase @Inject constructor(
```

Comment:
I'm not sure if this useCase is needed. It looks like convenience class to pass three informations at once to display this easier in the InfoScreen. I'm not sure if we should go this way, app should be lean and simple. We're introducing a model that's shared between the core and ui that couples them together. Let me know what you think, IMO we could just call `IsUserEnrolledForMobileAuthUseCase` and the other one useCase for Pushes that will come next without combining them in extra useCase.